### ISSUE TITLE
up能分享一下你的pytorch版本吗

### ISSUE THREAD

**Issue by Theyan277** [204313508/noname_llm#3]
conda创建的全新环境，py3.10, torch2.3.0+cu121, numpy2.0.1, 但是无法运行
报错The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

**Comment by 204313508** [comment]
懒人一键包采用的环境gpu版：
numpy==1.26.2
torch==2.2.1+cu118
torchaudio==2.2.1+cu118
torchvision==0.17.1+cu118
cpu版：
numpy==1.26.2
torch==2.1.1

**Comment by Theyan277** [comment]
好嘞 谢谢up。换了torch2.2.1，可以正常运行了，看来是版本问题:)
